### PR TITLE
feat(dashboard): Add tailwind, shadcn-ui config and react router

### PR DESCRIPTION
**Pull request by desiprisg**
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/dashboard/components.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "baseColor": "neutral",
+    "cssVariables": true,
+    "prefix": ""
+  },
+  "aliases": {
+    "components": "@/components",
+    "utils": "@/utils/ui",
+    "ui": "@/components/primitives",
+    "lib": "@/utils",
+    "hooks": "@/hooks"
```
Comment (LetItRock) [inline apps/dashboard/components.json:18]: `"ui": "@/components/primitives",` and `"lib": "@/utils",` are confusing to me.

Should we have instead: `"components/primitives": "@/components/primitives",` and `"lib": "@/lib", wdyt?

**Review by LetItRock — COMMENTED** [review]
lgtm, when will be ready let me know so I'll approve

File: apps/dashboard/components.json (hunk 1 above)
Comment (desiprisg) [reply apps/dashboard/components.json:18]: Left side needs to stay the same for the cli to work correctly.

Comment (LetItRock) [reply apps/dashboard/components.json:18]: ahh ok that's the structure that is forced by shadcn

File: novu.code-workspace [hunk 2]
```diff
@@ -110,7 +110,7 @@
     }
   ],
   "settings": {
-    "typescript.tsdk": "node_modules/typescript/lib",
+    "typescript.tsdk": "✨ novu root/node_modules/typescript/lib",
```
Comment (LetItRock) [inline novu.code-workspace]: what is that?

**Review by LetItRock — APPROVED** [review]

File: novu.code-workspace (hunk 2 above)
Comment (desiprisg) [reply novu.code-workspace]: I'll remove. I think vscode auto-added it when i used the ts version dropdown

**Pull request merged**